tests/python/frontend/pytorch/test_forward.py

```diff
@@ -4060,6 +4061,10 @@ def test_forward_pretrained_bert_base_uncased():
     print("TVM   top-1 id: {}, token: {}".format(tvm_pred_idx, tvm_pred_token))
 
 
+@pytest.mark.skipif(
+    platform.machine() == "aarch64",
+    reason="Currently failing on AArch64",
```

Comment:
Can you file a GitHub issue with the context on the skip and link it here? also just fyi I think we can merge all the skip test PRs since they have a similar theme

Reply:
Sure, we'll raise a tracking issue to investigate the fix for all tests being skipped in these PRs, namely this one, #12257, #12256.